transmart-rest-api-e2e/src/test/groovy/tests/rest/v2/DataExportSpec.groovy

```diff
@@ -0,0 +1,218 @@
+                                                  format  : 'TSV'
+                                          ]]),
+                ]),
+                statusCode: 403
+        ])
+        then: "I get an access error"
+        assert responseData.httpStatus == 403
+        assert responseData.type == 'AccessDeniedException'
+    }
+
+    def "run data export with 'Export' permission"() {
+        setUser(ADMIN_USERNAME, ADMIN_PASSWORD)
+        def newJobRequest = [
+                path      : "$PATH_DATA_EXPORT/job",
+                acceptType: contentTypeForJSON,
+        ]
+        def newJobResponse = post(newJobRequest)
+        def jobName = newJobResponse.jobName
+
+        when: "I run a newly created job asynchronously"
+        def runResponse = get([
+                path : "$PATH_DATA_EXPORT/$jobName/run",
+                query: ([
+                        setType : 'patient',
+                        ids     : 28835,
+                        elements: toJSON([[
+                                                  dataType: 'clinical',
+                                                  format  : 'TSV'
+                                          ],
+                                          [
+                                                  dataType: 'mrna',
+                                                  format  : 'TSV'
+                                          ]]),
+                ]),
+                acceptType: contentTypeForJSON,
+        ])
+
+        then: "The time at which the export will be run by the scheduler is returned"
+        assert runResponse != null
+        assert runResponse.jobName == jobName
+        assert runResponse.scheduledTime != null
+
+        when: "Check the status of the job"
+        int maxAttemptNumber = 10 // max number of status check attempts
+        def statusRequest = [
+                path      : "$PATH_DATA_EXPORT/$jobName/status",
+                acceptType: contentTypeForJSON,
+        ]
+        def statusResponse = get(statusRequest)
+
+        then: "Returned status is 'Completed'"
+        assert statusResponse != null
+        def status = statusResponse.jobStatus
+
+        // waiting for async process to end (increase number of attempts if needed)
+        for (int attempNum = 0; status != 'Completed' && attempNum < maxAttemptNumber; attempNum++) {
+            sleep(500)
+            statusResponse = get(statusRequest)
+            status = statusResponse.jobStatus
+        }
+
+        assert status == 'Completed'
+
+        when: "Try to download the file"
+        String fileName = "$TEMP_DIRECTORY/$jobName"+".zip"
+        def downloadRequest = [
+                path      : "$PATH_DATA_EXPORT/$jobName/download",
+                acceptType: contentTypeForZip,
+        ]
+        def downloadResponse = get(downloadRequest)
+
+        then: "ZipStream is returned"
+        assert downloadResponse != null
+        assert new File(fileName).isFile()
```

Comment:
IMHO no need to duplicate efforts in testing the content. It would be enough just to prove what is stated `ZipStream is returned`.